jkube-kit/helm/src/test/java/org/eclipse/jkube/kit/resource/helm/HelmServiceInstallIT.java

```diff
@@ -180,6 +180,6 @@ void whenDependencyReferencedDoesNotExist_thenThrowException() throws IOExceptio
     // When + Then
     assertThatIllegalStateException()
       .isThrownBy(() -> helmService.install(helmConfig))
-      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s/the-dependency not found", tempDir));
+      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s\\the-dependency not found", tempDir));
```

Comment:
```suggestion
      .withMessage(String.format("An error occurred while updating chart dependencies: directory %s%sthe-dependency not found", tempDir, File.separator));
```

Reply:
Hi Rohan,
corrected and pushed this change.